common-init.mk

```diff
@@ -55,10 +91,13 @@ PRODUCT_PACKAGES += \
     rild2.rc
 endif
 
-# Common init scripts
+ifeq ($(WIFI_DRIVER_BUILT),qca_cld3)
 PRODUCT_PACKAGES += \
-    init.qcom.modem.sh \
-    init.qcom.adspstart.sh \
-    init.qcom.cdspstart.sh \
-    init.qcom.ipastart.sh \
-    init.qcom.slpistart.sh
+    cnss-daemon.rc \
+endif
```

Comment:
You might want to take a look at these two lines :stuck_out_tongue_winking_eye:

Reply:
Nothing to see here... ;)